src/components/shared/ManageTokensModal.tsx

```diff
@@ -82,8 +82,7 @@ export default function ManageTokensModal({
     plural: true,
   })
 
-  // eslint-disable-next-line no-extra-boolean-cast
-  const redeemDisabled = !Boolean(hasOverflow)
+  const redeemDisabled = !hasOverflow
```

Comment:
wasn't 100% about going around and renaming these.

Reply:
na feels good 👍